proto/spacemesh/v1/global_state_types.proto

```diff
@@ -50,22 +50,46 @@ message AccountDataQueryRequest {
     uint32 offset = 3; // return results from offset
 }
 
+message ExecutionError {
+    uint32 code = 1; // error code
+    string message = 2; // error message
+}
+
+// From dev perspective we want to enable something like the following from the higher-level lang or rust code:
+// info([enty_name], [entry_text])
+// e.g. info("sig_verification", format!("%s : %d", str1, ui64))
+message LogEntry {
+    uint32 log_level = 1; // info, debug, trace, error, etc...
+    string context = 2; // function name e.g. withdraw()
+    string name = 3; // entry name
+    string entry = 4; // entry printed data
```

Comment:
in EVM/solidity, pretty sure this is a repeated set of field names and values rather than a single blob (per context/name)